task/rhacs-image-scan/3.71/README.md

```diff
@@ -0,0 +1,72 @@
+# StackRox/Red Hat Advanced Cluster Security Image Scan Task
+
+Scan an image for vulnerabilities and metadata against RHACS build and deploy lifecycle policies to validate a pipeline run using `roxctl`.
+
+This tasks allows you to return full vulnerability scan results for an image in JSON, CSV, or Pretty format.  It's a companion to the rhacs-image-check task.
```

Comment:
@dlbewley, the following Task already exists in the catalog with the name [stackrox-image-scan](https://github.com/tektoncd/catalog/tree/main/task/stackrox-image-scan/0.1) (I guess). If so, can we please bump the version of the existing task if there are any changes?

Reply:
@vinamra28 these tasks are developed specifically to target RHACS installations rather than the upstream solution. i am amenable to contributing enhancements to those tasks as appropriate in a subsequent PR.

Reply:
@dlbewley, will this task not work if StackRox is installed instead of RHACS?

Reply:
incorrect. it is not incompatible.